client/src/components/mapping/index.tsx

```diff
@@ -201,13 +169,25 @@ const MappingView = () => {
             <FhirMappingPanel />
           </div>
           <div id="exploration-panel">
-            {attribute && renderExistingRules()}
-            {attribute && renderTabs()}
-            {source.credential &&
-              resource &&
-              resource.primaryKeyTable &&
-              !attribute &&
-              renderTable()}
+            {attribute ? (
```

Comment:
can we avoid the ternary here ? might be easier to read

Reply:
What about now that I wrote a function?